### ISSUE TITLE
My Repo is no longer maintained

### ISSUE THREAD

**Issue by pythonInRelay** [pythonInRelay/ao#27]
Due to reasons beyond my control, I'm no longer maintaining this project at all. I also have no plans whatsoever to start doing so for the foreseeable future.

I recommend anyone interested in maintaining and improving it to fork my repo and work on it as normal. 

Thanks to everyone who worked on it.

If you really want to keep using Microsoft To-Do on Linux, I suggest harassing Microsoft for a native app for Linux.

**Comment by davidsmorais** [comment]
Since you stopped working on this, I've forked your repo, fixed a few bugs and rebranded it.

You can find a working version in this repo :point_down: 
https://github.com/davidsmorais/kuro

**Comment by 825i** [comment]
Yeah nah, I am not having any links to repos featuring pro Ukraine propaganda anywhere on my repos thanks.

You can keep your politics far away from any of the open source work I associate myself with. 🙄
